package.json

```diff
@@ -70,8 +70,8 @@
   },
   "dependencies": {
     "dct": "0.0.3",
+    "fftjs": "0.0.2-alpha.1",
```

Comment:
I'm glad it works - why don't you do a non-alpha release on fftjs and then upgrade to that here. Then we can merge! :)